start_ocean.sh

```diff
@@ -257,6 +258,18 @@ while :; do
             printf $COLOR_Y'Starting with local Duero node...\n\n'$COLOR_RESET
             printf $COLOR_Y'Starting without Secret Store...\n\n'$COLOR_RESET
             ;;
+        # connects you to Pacific ocean network
+        --local-pacific-node)
```

Comment:
this new flag needs to be documented in the respective part in the Readme

Reply:
Done. Thanks!